#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_manpage_pl_section.dpatch by  <javamaniac@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change the pl manpage section

@DPATCH@

--- swi-prolog-5.6.14.orig/src/pl.1.in.orig	2006-06-16 12:02:15.000000000 -0400
+++ swi-prolog-5.6.14/src/pl.1.in	2006-06-16 12:02:48.000000000 -0400
@@ -1,5 +1,5 @@
 .\" -*- nroff -*-
-.TH @PL@ 1L "Feb 8, 2006" "SWI-Prolog 5"
+.TH @PL@ 1 "Feb 8, 2006" "SWI-Prolog 5"
 .SH NAME
 @PL@ \- SWI-Prolog @PLVERSION@
 .SH SYNOPSIS
